source/api/accounts-multi.md

```diff
@@ -50,7 +50,8 @@ These functions return an object with a single method, `stop`.  Calling
 
 On the server, the callbacks get a single argument, the same attempt info
 object as [`validateLoginAttempt`](#accounts_validateloginattempt). On the
-client, no arguments are passed.
+client, an object is supplied containing only the error from the failed
```

Comment:
@abernix Yeah I wasn't 100% happy with the wording, I'm probably not going to become a top technical writer any time soon! I'll change it to your suggestion. Thanks :)